### PR TITLE
TFD Fixing Single Quotes in Strings (Mac)

### PR DESCRIPTION
**Pull request by time-killer-games**
lol it turned out to be more commits than I thought it would take.

3 words. Shut. Up.

Anyways this fixes a problem on Mac where if the user displayed a dialog with an apostrophe or single quote in the string the character would be ignored and taken out of the string because escaping wasn't done correctly by the original author of TFD. 

It's fixed! Yay! Let's get NAKED!!!!

### CODE REVIEW

**Comment by time-killer-games** [comment]
@JoshDreamland did I do the Makefile right and if so please merge. I don't use wildcards anymore in it so I don't know if the mention of wildcard should really be there but idk what to replace it with or if I should just remove it.

**Comment by time-killer-games** [comment]
why is not passing? wow

**Comment by RemoveRusky** [comment]
ci crapped out I restarted job for you

File: ENIGMAsystem/SHELL/Widget_Systems/TinyFileDialogs/Makefile [hunk 1]
```diff
@@ -1,2 +1,9 @@
 include $(SHARED_SRC_DIR)/tinyfiledialogs/Makefile
-SOURCES += $(wildcard Widget_Systems/TinyFileDialogs/*.cpp)
+
+ifeq ($(TARGET-PLATFORM), MacOSX)
+	SOURCES += $(wildcard Widget_Systems/TinyFileDialogs/dialogs-macos.cpp)
```
Comment (RemoveRusky) [inline ENIGMAsystem/SHELL/Widget_Systems/TinyFileDialogs/Makefile]: wildcard is for when you have * you can just use SOURCES += Widget_Systems/TinyFileDialogs/dialogs-macos.cpp here

**Pull request merged**